Tests/ClientTests/Frontend/Home/TopSites/TopSitesDimensionTests.swift

```diff
@@ -12,7 +12,7 @@ class TopSitesDimensionTests: XCTestCase {
     func testSectionDimension_portraitIphone_defaultRowNumber() {
         let subject = createSubject()
         let trait = MockTraitCollection()
-        let interface = TopSitesUIInterface(isLandscape: false, isIphone: true, trait: trait)
+        let interface = TopSitesUIInterface(isLandscape: false, isIphone: true, trait: trait, availableWidth: 390)
```

Comment:
Same comment for width, maybe adding something that implicitly tells us that this means. For each (390, 844, 820, 1180, 320, 375)